components/resources/brave_news_strings.grdp

```diff
@@ -69,6 +69,18 @@
   <message name="IDS_BRAVE_NEWS_SUGGESTIONS_SUBTITLE" desc="Subtitle of the suggested sources section">
     Some sources you might like. Suggestions are anonymous and matched only on your device.
   </message>
+  <message name="IDS_BRAVE_NEWS_ERROR_HEADING" desc="Message heading for when there is a Brave News error">
+    Oops…
+  </message>
+  <message name="IDS_BRAVE_NEWS_ERROR_MESSAGE" desc="Message body for when there is a Brave News error">
+    Brave News is experiencing some issues. Try again.
```

Comment:
Try again, sounds kinda brusque. Maybe `Please try again later`?

Reply:
Well, they might want to try immediately not later if it was a network issue (which it probably was) 😆 . Really we should have a button to refresh on this card and its text would be "Try again". But it's likely floating above this message somewhere.

Reply:
Moved to a button with "Try Again" label